libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1006,6 +1006,16 @@ AP_InertialSensor::init(uint16_t loop_rate)
 #endif
 
 #if AP_INERTIALSENSOR_HARMONICNOTCH_ENABLED
+
+#if APM_BUILD_TYPE(APM_BUILD_Heli)
+    // Throttle tracking does not make sense with heli because "throttle" is actually collective position in AP_MotorsHeli
+    for (auto &notch : harmonic_notches) {
+        if (notch.params.enabled() && notch.params.tracking_mode() == HarmonicNotchDynamicMode::UpdateThrottle) {
+            AP_BoardConfig::config_error("Throttle notch unavailable with heli");
```

Comment:
Upon further though, I think it might have been better if this had been made an arming check.  With a config_error the main loop won't run and the user will find the HUD doesn't update.  Sadly this will only happen the next time that they reboot the board so they may not remember that they changed the harmonic notch method